shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -581,7 +593,7 @@ - (void)blockGesture {
       // FlutterPlatformViewGestureRecognizersBlockingPolicyEager, but we should try it if a similar
       // issue arises for the other policy.
       if (@available(iOS 18.2, *)) {
-        if ([self.embeddedView isKindOfClass:[WKWebView class]]) {
+        if ([self containsWebView:self.embeddedView]) {
           [self removeGestureRecognizer:self.delayingRecognizer];
           [self addGestureRecognizer:self.delayingRecognizer];
```

Comment:
Will this cause problems in the views they are gesturing into that are not WKWebViews?

Reply:
I think it is possible. That's something that we need to watch out for. But I highly doubt if it is gonna cause real issues

Reply:
We are talking about an odd of combining: 

1. having a setup with webview + another non-web-view subview that also handles gesture
2. having a flutter overlay layer on top of this non-web-view subview

Then we will have this non-web-view subview receive the extra touch when it's not supposed to. 

Then even when this happens, I don't think it is a huge problem.